packages/wallets/src/types.ts

```diff
@@ -72,3 +72,6 @@ export namespace TDisplayBalance {
 }
 
 export type TGenericSizes = '2xl' | '2xs' | '3xl' | '3xs' | '4xl' | '5xl' | '6xl' | 'lg' | 'md' | 'sm' | 'xl' | 'xs';
+
+export type TWalletLandingCompanyName = Extract<THooks.MT5AccountsList['landing_company_short'], 'malta' | 'svg'>;
```

Comment:
@heorhi-deriv  i think for now this is okay. at least if it doesnt exist from the api anymore, it will throw a TS error and we can actually clean it up